server/service/src/sync/synchroniser.rs

```diff
@@ -285,6 +285,11 @@ impl Synchroniser {
             .trigger_requisition_transfer_processors();
         ctx.processors_trigger.trigger_invoice_transfer_processors();
 
+        if CentralServerConfig::is_central_server() {
+            ctx.processors_trigger
+                .trigger_contact_form_to_email_processors();
+        }
```

Comment:
This seemed like the safest place to trigger - after OMS central does a sync cycle... originally thought it would be in the `sync_on_central` code, after a push, but then we could also receive support requests/feedback from central site.

Does this seem right to you?

Reply:
I think this looks good, however, I think we should possibly move the central server check into the processor? That way if we wanted to trigger somewhere else as well we wouldn't need to repeat the logic?

Reply:
[9eecec7](https://github.com/msupply-foundation/open-msupply/pull/5936/commits/9eecec79c7c962d1e1081150ab9e58f943162aef)